base_fontawesome/README.rst

```diff
@@ -0,0 +1,44 @@
+.. image:: https://img.shields.io/badge/licence-lgpl--3-blue.svg
+   :target: http://www.gnu.org/licenses/LGPL-3.0-standalone.html
+   :alt: License: LGPL-3
+
+Base fontawesome
+================
+
+Provide up to date `Fontawesome <http://fontawesome.io/>`_ resources.
+
+Current version: 4.7.0.
+
+
```

Comment:
Add usage section indicating how to use this (or you just need to work the same way as standard? - just indicate it then).

Reply:
@pedrobaeza better now?